new-pipeline/src/dashboards.js

```diff
@@ -227,8 +227,8 @@ function loadStateFromUrlAndCookie() {
     var aggregates = pageState.aggregates.split("!")
       .filter(function (v) {
         return ["5th-percentile", "25th-percentile", "median",
-            "75th-percentile", "95th-percentile", "mean", "submissions"].indexOf(v) >= 0 ||
-          v.startsWith("bucket-");
+            "75th-percentile", "95th-percentile", "mean", "submissions"].indexOf(v) >= 0
+            ||(v.trim() != "");
```

Comment:
Need a space after `||`, or you can leave it at the end of the previous line.

Reply:
The changes made here is actually done to make the old url's to also work so i have added the change of checking for only non-empty values and later on check the validity of the values in the savestateToCookieandUrl methods